diff --git a/node_modules/expo-modules-autolinking/build/platforms/apple/apple.js b/node_modules/expo-modules-autolinking/build/platforms/apple/apple.js
index 47487c0..58a6069 100644
--- a/node_modules/expo-modules-autolinking/build/platforms/apple/apple.js
+++ b/node_modules/expo-modules-autolinking/build/platforms/apple/apple.js
@@ -113,7 +113,7 @@ async function generatePackageListFileContentAsync(modules, className, entitleme
  * but only these that are written in Swift and use the new API for creating Expo modules.
  */

-import ExpoModulesCore
+public import ExpoModulesCore
 ${generateCommonImportList(swiftModules)}
 ${generateDebugOnlyImportList(debugOnlySwiftModules)}
 @objc(${className})
@@ -137,13 +137,13 @@ ${generateReactDelegateHandlers(reactDelegateHandlerModules, debugOnlyReactDeleg
 `;
 }
 function generateCommonImportList(swiftModules) {
-    return swiftModules.map((moduleName) => `import ${moduleName}`).join('\n');
+    return swiftModules.map((moduleName) => `public import ${moduleName}`).join('\n');
 }
 function generateDebugOnlyImportList(swiftModules) {
     if (!swiftModules.length) {
         return '';
     }
-    return (wrapInDebugConfigurationCheck(0, swiftModules.map((moduleName) => `import ${moduleName}`).join('\n')) + '\n');
+    return (wrapInDebugConfigurationCheck(0, swiftModules.map((moduleName) => `public import ${moduleName}`).join('\n')) + '\n');
 }
 function generateModuleClasses(classNames, debugOnlyClassName) {
     const commonClassNames = formatArrayOfClassNames(classNames);
